src/rust/dbv2/migrations/2023-09-04-113144_postgrest/up.sql

```diff
@@ -0,0 +1,29 @@
+-- Your SQL goes here
+CREATE ROLE web_anon nologin;
```

Comment:
Maybe drop the `anon` role (and do other undos) that I made in the previous migration.